mne/time_frequency/multitaper.py

```diff
@@ -330,6 +332,7 @@ def psd_array_multitaper(
     adaptive=False,
     low_bias=True,
     normalization="length",
+    remove_dc=True,
```

Comment:
I thought @larsoner [suggested a default value of `False`](https://github.com/mne-tools/mne-python/issues/11767#issuecomment-1615038293) here?  Though it would seem like `remove_dc=True` (as you've done) is the backwards-compatible way.  @larsoner can you clarify: do you consider current practice of silently removing DC to be a bug, and hence default should be `False` and this would be considered a bugfix?

Reply:
checking in what is the consensus about this? @drammock @larsoner

Reply:
updated considering that the default behavior should be remove_dc=False. NOT backward-compatible